app/code/Magento/Tax/Model/Calculation/RateRepository.php

```diff
@@ -270,7 +273,7 @@ private function validate(\Magento\Tax\Api\Data\TaxRateInterface $taxRate)
             );
         }
 
-        if (!\Zend_Validate::is(trim($taxRate->getCode()), 'NotEmpty')) {
+        if (!\Zend_Validate::is(trim($taxRate->getCode() ?? ''), 'NotEmpty')) {
```

Comment:
as for me ` if ($taxRate->getCode() === null || !\Zend_Validate::is(trim($taxRate->getCode()), 'NotEmpty')) {` is better

Reply:
@xmav I changed, thanks for your help!